packages/ag-charts-website/src/content/docs/themes/_examples/stock-themes/main.ts

```diff
@@ -1,24 +1,13 @@
 import { AgChartOptions, AgChartTheme, AgChart } from 'ag-charts-community'
+import { getData } from './data';
 
 const options: AgChartOptions = {
   container: document.getElementById('myChart'),
   theme: 'ag-default-dark',
-  autoSize: true,
```

Comment:
Did you mean to delete `autoSize` here?

Reply:
Yes, it's the default so shouldn't be necessary?

Reply:
I've removed all redundant `autoSize: true` from examples to cleanup generally.